lib/Listener/RegisterOperationsListener.php

```diff
@@ -27,23 +27,30 @@
 
 use OCA\FlowNotifications\AppInfo\Application;
 use OCA\FlowNotifications\Flow\Operation;
+use OCP\App\IAppManager;
 use OCP\EventDispatcher\Event;
 use OCP\EventDispatcher\IEventListener;
 use OCP\Util;
 use OCP\WorkflowEngine\Events\RegisterOperationsEvent;
+use Psr\Log\LoggerInterface;
 
 /**
  * @template-implements IEventListener<Event|RegisterOperationsEvent>
  */
 class RegisterOperationsListener implements IEventListener {
 	public function __construct(
-		protected readonly Operation $operation
+		protected readonly Operation $operation,
+		protected IAppManager $appManager,
+		private readonly LoggerInterface $logger,
 	) {
 	}
 
 	public function handle(Event $event): void {
 		if (!$event instanceof RegisterOperationsEvent) {
 			return;
+		} elseif (!$this->appManager->isEnabledForUser('notifications')) {
+			$this->logger->error('Failed to register `flow_notifications` app. This could happen due to the `notifications` app isn\'t installed or enabled.', ['app' => 'flow_notifications']);
```

Comment:
```
$this->logger->error('Failed to register `flow_notifications` app. This could happen due to the `notifications` app not being installed or enabled.', ['app' => 'flow_notifications']);
```

🙈

Reply:
@falghamdi125 he has a point there ^